controller/news.controller.js

```diff
@@ -52,3 +53,15 @@ exports.postedCommentById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getPatchedArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  const votes = req.body;
+  patchedArticleById(article_id, votes)
+    .then((updatedArticles) => {
+      res.status(201).send({ articles: updatedArticles });
```

Comment:
Change this to a key of 'article' as you are only expecting a single article object to be returned.